### DISCUSSION TITLE
Has anyone tried running whisper on a Raspberry Pi Zero W? It only has 512MB of RAM so I am concerned on how quickly it would run.

### DISCUSSION THREAD

**Discussion by KonradLach** [openai/whisper#50] [Q&A]
I am trying to use whisper and a Pi to add more languages to work with Alexa. I have a spare Pi Zero W laying around and don't want to buy a Pi 4.

**Comment by werelord** [comment]
I'm not sure if it will run at all..

I've been playing with this using cuda on my GTX 970 (old, I know), and I can't run anything bigger than the small model; at the medium model I get an OOM exception; it tried allocating more memory than the card had available.  I would suspect that the Pi would have the same issue (although I don't know for certain, YMMV)

**Comment by openaudible** [comment]
Here is someone who has it working on a pi 
https://www.ikkaro.com/en/control-voz-pc-raspberrypi/

**Comment by ArtyomZemlyak** [comment]
Check this issue
https://github.com/ggerganov/whisper.cpp/issues/7

**Comment by nyadla-sys** [comment]
On a Raspberry Pi 4, the Whisper C/C++ model below was run, and it takes 15 seconds to execute one inference.
https://github.com/ggerganov/whisper.cpp
